test/mongo_test.exs

```diff
@@ -124,6 +124,24 @@ defmodule Mongo.Test do
       Mongo.find(c.pid, coll, %{}, sort: [foo: -1], batch_size: 2, limit: 2) |> Enum.to_list
   end
 
+  test "find_one", c do
+    coll = unique_name()
+
+    assert nil == Mongo.find_one(c.pid, coll, %{})
+
+    assert {:ok, _} = Mongo.insert_one(c.pid, coll, %{foo: 42, bar: 1})
+
+    assert %{"foo" => 42} = Mongo.find_one(c.pid, coll, %{}, batch_size: 1)
+
+    assert {:ok, _} = Mongo.insert_one(c.pid, coll, %{foo: 43})
+
+    # we should still get one
+    assert %{"foo" => 42} = Mongo.find_one(c.pid, coll, %{}, batch_size: 1)
+
+    assert %{"foo" => 43} != Mongo.find_one(c.pid, coll, %{}, batch_size: 1)
+
+  end
```

Comment:
Since the asserts are still single line we can remove the empty lines around them.